ultra/docker/remote.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# An example to execute this script:
+# source remote.sh <user name> <password> <server name> <container name>
+# source remote.sh z84216771 abcd1234 GX3 ultratest
+
+# Servers
+# CX3 = 10.193.241.233
+# CX4 = 10.193.241.234
+# Compute-4 = 10.193.192.17
+# Compute-11 = 10.193.192.113
+# GX3 = 10.193.241.239
+
+# Arguments
+USER=$1       #Username, e.g., z84216771
+PASS=$2       #Password, e.g., abcd1234
+SERVER=$3     #Server, e.g., GX3
+CONTAINER=$4  #Container name, e.g., ultratest
+
+case $SERVER in
+    "CX3")
+        ADD=10.193.241.233
+        DST=/data/research
+        GPUFLAG=""
+        ;;
+    "CX4")
+        ADD=10.193.241.234
+        DST=/data/research
+        GPUFLAG=""
+        ;;
+    "Compute-4")
+        ADD=10.193.192.17
+        DST=/data/$USER
+        GPUFLAG="--runtime=nvidia"        
+        ;;
+    "Compute-11")
+        ADD=10.193.192.113
+        DST=/data/$USER
+        GPUFLAG="--runtime=nvidia"
+        ;;
+    "GX3")
+        ADD=10.193.241.239
+        DST=/data/research
+        GPUFLAG="--gpus=all"
+        ;;
+    *)
+        echo "Unknown server"
+        return 5
+        ;;
```

Comment:
This should not be in here I think

Reply:
@Adaickalavan

Reply:
I removed the docker folder from git

Reply:
The documentation for code deployment in our internal servers, has been uploaded into our own GitLab. I have forwarded the link in our chat group. Hence, please feel free to remove the above file.